packages/discovery-provider/src/api/v1/tracks.py

```diff
@@ -355,7 +355,7 @@ def get(self):
             response, status = success_response(tracks)
             return marshal(response, full_track_response), status
         else:
-            tracks = [extend_track(track) for track in tracks]
+            tracks = list(map(extend_track, tracks))
```

Comment:
Cue an argument about being "pythonic".

Reply:
i literally replaced all of these with list comprehensions so i could pass more args w/o lambdas but didn't need that once i moved to query helpers so i manually went to undo them, guess this one was a comprehension before woops lol